src/api/goods.ts

```diff
@@ -3,17 +3,22 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
+export async function getAll(): Promise<Good[]> {
+  const response = await fetch(API_URL);
```

Comment:
I think you need to check if response status is ok like this before returning it

if (!response.ok) {
...
}

Reply:
I'm not sure about that🤔. 
the result of this async function (promise) is used in external code further and errors are processed by callback method catch(). And as I understood the error parameter value in the catch() method is an error which will be await if the promo is rejected.